osu.Game/Rulesets/Edit/BeatmapVerifierContext.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Bindables;
+using osu.Game.Beatmaps;
+
+namespace osu.Game.Rulesets.Edit
+{
+    /// <summary>
+    /// Represents the context provided by the beatmap verifier to the checks it runs.
+    /// Contains information about what is being checked and how it should be checked.
+    /// </summary>
+    public class BeatmapVerifierContext
+    {
+        /// <summary>
+        /// The working beatmap instance of the current beatmap.
+        /// </summary>
+        public readonly IWorkingBeatmap WorkingBeatmap;
+
+        /// <summary>
+        /// The difficulty level which the current beatmap is considered to be.
+        /// </summary>
+        public readonly Bindable<DifficultyRating> InterpretedDifficulty;
```

Comment:
If consumers of this context are not intending to handle changes in this bindable (it doesn't look like they would/should) then this should just be copied out of the bindable for clarity. Also avoids the `GetBoundCopy` being passed into a new instance, which is something we generally don't do.